config/crd/kustomization.yaml

```diff
@@ -8,12 +8,12 @@ resources:
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-#- patches/webhook_in_mirrorpeers.yaml
+- patches/webhook_in_mirrorpeers.yaml
 #+kubebuilder:scaffold:crdkustomizewebhookpatch
 
 # [CERTMANAGER] To enable webhook, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
-#- patches/cainjection_in_mirrorpeers.yaml
+- patches/cainjection_in_mirrorpeers.yaml
```

Comment:
Is this specific to cert-manager or is it generic? Let's not use cert-manager for now.